### PR TITLE
Adding repo and area owners to the codeowners file

### PR DESCRIPTION
**Pull request by iamgusain**
Adding repo code and area owners as per the list from [here](https://microsoft.sharepoint.com/teams/ADAL/_layouts/15/Doc.aspx?sourcedoc={0339bdd6-b5e5-4d94-b4c9-ea47127f5023}&action=edit&wd=target%28Platforms%2FAndroid.one%7C28086b2a-f539-bc47-b229-1610fc61c903%2FCode%20Owners%7C83d87699-1bfc-46f7-b164-0df4c71a1d16%2F%29)

Once this change is merged, I will enable the branch protection setting to require codeowners approval

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,37 @@
...
+
+# Repo Code Owners
+#--------------------
+
+*       @AdamBJohnsonx @iambmelt @rpdome @shahzaibj @shoatman
+
+
+# Area Owners
+#---------------------
+
```
Comment (iambmelt) [inline CODEOWNERS:13]:
```suggestion
# AuthSchemes
/common/src/main/java/com/microsoft/identity/common/internal/authscheme @iambmelt @AdamBJohnsonx
```

Comment (iambmelt) [inline CODEOWNERS:13]:
```suggestion
# Broker
/common/src/main/java/com/microsoft/identity/common/internal/broker @rpdome
```

Comment (iambmelt) [inline CODEOWNERS:13]:
```suggestion
# Commands
/common/src/main/java/com/microsoft/identity/common/internal/commands @shoatman @iambmelt @AdamBJohnsonx 
```

Comment (iambmelt) [inline CODEOWNERS:13]:
```suggestion
# DTOs
/common/src/main/java/com/microsoft/identity/common/internal/dto @iambmelt @AdamBJohnsonx 
```

Comment (iambmelt) [inline CODEOWNERS:13]:
```suggestion
# Networking
/common/src/main/java/com/microsoft/identity/common/internal/net @AdamBJohnsonx 
```

Comment (iambmelt) [inline CODEOWNERS:13]:
```suggestion
# Migration
/common/src/main/java/com/microsoft/identity/common/internal/migration @iambmelt @rpdome @shahzaibj 
```

**Review by AdamBJohnsonx — APPROVED** [review]
LGTM, though we need to get more code owners.

**Review by iambmelt — APPROVED** [review]

**Pull request merged**